ios/SourceModule.m

```diff
@@ -2,8 +2,15 @@
 
 @interface RCT_EXTERN_REMAP_MODULE(SourceModule, SourceModule, NSObject)
 
-RCT_EXTERN_METHOD(initWithConfig:(NSString *)nativeId drmNativeId:(NSString *)drmNativeId config:(nullable id)config)
-RCT_EXTERN_METHOD(initWithAnalyticsConfig:(NSString *)nativeId drmNativeId:(NSString *)drmNativeId config:(nullable id)config analyticsSourceMetadata:(nullable id)analyticsSourceMetadata)
+RCT_EXTERN_METHOD(initWithConfig:(NSString *)nativeId
+                  drmNativeId:(NSString *)drmNativeId
+                  config:(nullable id)config
+                  sourceRemotePlaybackConfig:(id)remotePlayback)
+RCT_EXTERN_METHOD(initWithAnalyticsConfig:(NSString *)nativeId
+                  drmNativeId:(NSString *)drmNativeId
+                  config:(nullable id)config
+                  sourceRemotePlaybackConfig:(id)remotePlayback
+                  analyticsSourceMetadata:(nullable id)analyticsSourceMetadata)
```

Comment:
Nit: Seems like the convention to align with `:` is not a thing in this file. However, looking at the rest of the file, it seems like the indentation is one tab size and not based on `(`.

Reply:
fixed in 85c58e781a68f459db5d640ccaea2bcd48c2985c